src/html/popup.html

```diff
@@ -124,16 +124,12 @@ <h4  id="tip" class="invisible">Tip: Click on an individual version to view it i
           <div id="declaredlicenses">
             <tr>
               <td>
```

Comment:
Was it difficult to add <tr> elements instead and to build all of the text below this in popup.js?

Reply:
Yeah, it's a combined table. I didn't want to split it apart into two tables (because lol css and hell), so I went with this for now.

Reply:
Context: I believe @ctownshend is rewriting things in React (or I remember reading that)